src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -2059,4 +2059,30 @@ Please specify one of them using the --source option to proceed.</value>
   <data name="InstallerDownloadCommandProhibited" xml:space="preserve">
     <value>Installer is prohibited from being downloaded for later offline installation. </value>
   </data>
+  <data name="ResumeCommandLongDescription" xml:space="preserve">
+    <value>Resumes execution of a previously saved command by passing in the unique guid identifier of the saved command. This is used to resume an executed command that may have been terminated due to a reboot.</value>
+  </data>
+  <data name="ResumeCommandShortDescription" xml:space="preserve">
+    <value>Resumes execution of a previously saved command.</value>
+  </data>
+  <data name="ResumeGuidArgumentDescription" xml:space="preserve">
+    <value>The unique guid identifier of the saved state to resume</value>

```

Comment:
`unique guid identifier` is redundant. Suggest switching to the full definition or removing `guid`
```suggestion
    <value>The globally unique identifier (guid) of the saved state to resume</value>
    <comment>{Locked="guid"} 'guid' is the short name form of a globally unique identifier.</comment>
```

```suggestion
    <value>The unique identifier of the saved state to resume</value>
```